server/api/userOrderRoutes.js

```diff
@@ -0,0 +1,148 @@
+const router = require('express').Router()
+const {
+  UserOrder,
+  Product,
+  OrderProduct,
+  Brand,
+  Review
+} = require('../db/models')
+const Sequelize = require('sequelize')
+module.exports = router
+
+router.get('/', async (req, res, next) => {
+  try {
+    const userOrder = await UserOrder.findAll({
+      include: [
+        {
+          model: OrderProduct
+        }
+      ]
+    })
+    res.status(200).send(userOrder)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.get('/cart/:userId', async (req, res, next) => {
+  const userId = req.params.userId
+  try {
+    const userOrder = await UserOrder.findAll({
+      where: {userId, isCheckedOut: false},
+      include: [
+        {
+          model: OrderProduct,
+          attributes: {
+            include: [
+              [Sequelize.literal('SUM(price * quantity)'), 'totalPrice']
+            ]
+          },
+
+          include: [
+            {
+              model: Product,
```

Comment:
I wonder if we're going to have to also do an include on the Image model since Product no longer has an imageURL field

Reply:
Good point, added that